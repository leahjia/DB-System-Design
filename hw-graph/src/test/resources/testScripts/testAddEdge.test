# University Map new Edge creation testing file.
# Tests that a single Edge can be created/added to the university map

# Create a graph
CreateGraph map1

# Add a pair of edges
AddEdge map1 n1 n2 e1
AddEdge map1 n2 n1 e2