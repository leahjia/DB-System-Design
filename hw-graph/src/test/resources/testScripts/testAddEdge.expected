# UnivMap adding edges testing file.
# Tests that edges can be added to the university map
# with new nodes also automatically added to the map

# Create a graph
created graph map1

# Add a pair of edges
added edge e1 from n1 to n2 in map1
added edge e2 from n2 to n1 in map1
